src/inference/tests/functional/caching_test.cpp

```diff
@@ -126,6 +126,8 @@ class MockExecutableNetwork : public IExecutableNetworkInternal {
     MOCK_METHOD0(CreateInferRequest, IInferRequestInternal::Ptr());
     MOCK_CONST_METHOD0(GetInputsInfo, ConstInputsDataMap());
     MOCK_CONST_METHOD0(GetOutputsInfo, ConstOutputsDataMap());
+    MOCK_CONST_METHOD0(getInputs, const std::vector<std::shared_ptr<const ov::Node>>&());
```

Comment:
Why do you need this change?

Reply:
Since Hetero CompiledModel restores inputs/outputs from compiled submodels, I need inputs/outputs information from submodel (it is done according to `GetInputsInfo`/`GetOutputsInfo`)

Reply:
https://github.com/openvinotoolkit/openvino/pull/18429/files#r1259446000